using_images/other_images/jenkins.adoc

```diff
@@ -293,6 +293,16 @@ The default maven agent pod uses `docker.io/openshift/jenkins-agent-maven-35-cen
 the CentOS or RHEL version of the Jenkins image. This variable must be set before Jenkins starts
 the first time for it to have an effect.
 
+* `JENKINS_UC_INSECURE`
++
+Determines whether Jenkins plugins downloads are allowed if the Jenkins Update Center repository 
+uses an invalid SSL certificate. This could be the case if a self hosted repository 
+using self-signed certificate with an unknow CA is used or if an enteprise proxy  
+performs man-in-the-middle interceptions. This variable applies to plugins downloads which may 
+occur during Jenkins image build or if an extension of the Jenkins image is built or if you 
+run the jenkins image and leverage one of the options to download additional plugins 
+(use of s2i whith plugins.txt or use of `INSTALL_PLUGINS` environment variable.
+To enable, set to `true`.
```

Comment:
missing ‘)’

Reply:
Fixed

Reply:
@akram After setting INSTALL_PLUGINS with specified version plugins, the jenkins s2i build didn't install the versioned plugins.
Could you make the INSTALL_PLUGINS  usage more clear? Thanks~
$ oc build-logs  jenkins-master-2  | grep "openshift-sync"
Command "build-logs" is deprecated, use oc logs build/<build-name>
STEP 3: ENV OPENSHIFT_BUILD_NAME="jenkins-master-2" OPENSHIFT_BUILD_NAMESPACE="xiu" OPENSHIFT_BUILD_SOURCE="https://github.com/xiuwang/jenkins-example.git" OPENSHIFT_BUILD_REFERENCE="Jenkins_uc_insecure" OPENSHIFT_BUILD_COMMIT="e6941401a68da0016e761fe98fdf1681132e551f" JENKINS_UC="https://***" JENKINS_UC_INSECURE="true" INSTALL_PLUGINS="openshift-sync:1.0.37"
openshift-sync:1.0.38